layouts/partials/home.html

```diff
@@ -18,7 +18,8 @@ <h1 class="greeting">Hi, I am {{ if .Site.Params.author.name }}{{ .Site.Params.a
       <li>{{ . }}</li>
       {{ end }}
     </ul>
```

Comment:
I wasn't sure how to sort by `weight` and also filter for only `enable=true`. I tried something similar to this using `where` but wasn't able to get it to work.
https://discourse.gohugo.io/t/solved-sort-and-where-in-a-range-site-data/8225/2